packages/web-components/fast-foundation/src/data-grid/data-grid.spec.md

```diff
@@ -121,6 +211,21 @@ Can be either "none", "default" or "sticky" (the `GeneratHeaderOptions` enum). A
 - `grid-template-columns`  
 String that gets applied to the the css gridTemplateColumns attribute of child rows. Corresponds to the [grid-template-columns css attribute](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template-columns)
 
+- `selection-mode`
+Sets how the grid handles selection.
+"none" - The default.  The grid does not enable selection.
+"single-row" - Single row elements are selectable.
+"multi-row" - Multiple rows can be selected.
+
+- `click-select`
```

Comment:
This was renamed, and I suggested further renaming it below.

Reply:
updated with "selection-behavior"